src/components/editor/ResourceTemplateForm.jsx

```diff
@@ -58,10 +58,14 @@ export class ResourceTemplateForm extends Component {
     })
   }
 
+  cardRow = () => {
+
+  }
+
   renderComponentForm = () => (
     <div>
       <form onSubmit={e => e.preventDefault()}>
-        <div className="ResourceTemplateForm">
+        <div className="ResourceTemplateForm row">
```

Comment:
Is ResourceTemplateForm necessary? Should it be an id?

Reply:
Removed, not necessary.